edk2toolext/omnicache.py

```diff
@@ -145,6 +155,10 @@ def _ConvertOmnicache(self):
                 logging.info("Removing remote {0} with duplicate URL {1}".format(name, url))
                 RunCmd("git", "remote remove {0}".format(name), workingdir=self.path)
                 RunCmd("git", "config --local --unset omnicache.{0}.displayname".format(name), workingdir=self.path)
+        # setup fetch.parallel to a reasonable default
```

Comment:
Why is this default reasonable

Reply:
Will add a more informative comment; that was just the git verbiage.